Minor chest dialog changes

Cheerfulbull opened the pull request:
Close #9024 by centring the XP icon and text and changing the XP message to "XP Needed". One problem: the XP message, when centred, becomes very cramped (comes very close to the edge). What would be the best way to fix this? 
Thank you.

oleg-derevenetz commented:
Hi @Cheerfulbull I suggest we limit the scope of this PR to #9025 and create a separate PR for #9024. The idea with the phrase "XP needed:" is questionable in general, because:

* The string is getting longer without explicit necessity;
* It is in English that there is a convenient two-letter abbreviation "XP", meaning "experience". In other languages, there may not be such a convenient short abbreviation, that is, we get a localization issue out of the blue.

Cheerfulbull commented:
> Hi @Cheerfulbull I suggest we limit the scope of this PR to #9025 and create a separate PR for #9024. The idea with the phrase "XP needed:" is questionable in general, because:
> 
> * The string is getting longer without explicit necessity;
> * It is in English that there is a convenient two-letter abbreviation "XP", meaning "experience". In other languages, there may not be such a convenient short abbreviation, that is, we get a localization issue out of the blue.

Alright, thank you, I updated the code and removed the text change, the updated PR description would then be:
>Close #9025 by centring the XP icon and text.

oleg-derevenetz commented:
If I understand correctly what the issue is, I suppose @LeHerosInconnu will say they're still not centered. With this PR:

![chest](https://github.com/user-attachments/assets/7f2d1580-8446-4c36-8da2-4617f8dc79f9)

@LeHerosInconnu what do you think?

LeHerosInconnu commented:
Hello @Cheerfulbull and @oleg-derevenetz,

> If I understand correctly what the issue is, I suppose @LeHerosInconnu will say they're still not centered. With this PR:
> 
> ![chest](https://github.com/user-attachments/assets/7f2d1580-8446-4c36-8da2-4617f8dc79f9)
> 
> @LeHerosInconnu what do you think?

Effectively, you can see even with the naked eye that graphics and texts are not centered horizontally in relation to the respective buttons. :)

![Centering chest elements 01](https://github.com/user-attachments/assets/1d4aa808-cf02-4d67-90c4-a3481a2f34ae)

LeHerosInconnu commented:
Hello @ihhub,

> Hi @LeHerosInconnu , does this change fixes the original problem?

Okay, I'll test it when the conflicts will be resolved.

ihhub commented:
@LeHerosInconnu , actually can you please check the latest snapshot? I believe we might have fixed it.

ihhub commented:
Closing this pull request as the original problem was fixed in https://github.com/ihhub/fheroes2/pull/9103
